BryantUniversity/BryantUniversity/Controllers/FacultyController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BryantUniversity.Data;
+using BryantUniversity.Models;
+using BryantUniversity.Models.Repo;
+using BryantUniversity.ViewModels;
+using System.Web.Mvc;
+
+namespace BryantUniversity.Controllers
+{
+    public class FacultyController : Controller
+    {
+        private Context context;
+
+        public FacultyController()
+        {
+            context = new Context();
+        }
+
+        // GET: Faculty
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        public ActionResult Create()
+        {
+            var userViewModel = new UserViewModel();
+            return View(userViewModel);
+        }
+
+        [HttpPost]
+        public ActionResult Create(UserViewModel userViewModel)
+        {
+            if (ModelState.IsValid)
+            {
+                string hashedPassword = BCrypt.Net.BCrypt.HashPassword(userViewModel.Password);
+                User user = new User(userViewModel.Email, hashedPassword, userViewModel.Name);
+                var repository = new UserRepo(context);
```

Comment:
I mentioned this before, but any time `context` is used, it should be within a `using` block to ensure that it gets disposed correctly. It's not a big deal for small applications like this, but it's good to develop that habit to prevent memory leaks. It's also another reason to use a dependency injection container; Unity will handle disposing any dependencies automatically.